app/components/publications/publication/case/contact-person-edit-modal.js

```diff
@@ -48,7 +48,7 @@ export default class PublicationsPublicationCaseContactPersonEditModalComponent
       },
     });
     this.organizations = organizations.filter((org) => org.name);
-    this.organizations = organizations.sortBy('name');
+    this.organizations = this.organizations.sortBy('name');
```

Comment:
Vraag= moet je die op deze manier setten om de tracked te triggeren of je kan gewoon
`this.organizations.sortBy('name');` zonder de `this.organizations =` ?

Reply:
sortBy sorteerd een kopie van de array. Heb ik net geleerd ;~)